javascript/javascript1/week2/script.js

```diff
@@ -3,13 +3,39 @@
 
 // Flight booking fullname function
 
-function getFullname(firstname, surname) {
-    return firstname + " " + surname;
-};
 
-console.log(getFullname("Anastasiia", "Yaitska"));
+// 1. first part
+
+// function getFullname(firstname, surname) {
+ 
+//     return firstname + " " + surname;
+// };
+
+// console.log(getFullname("Anastasiia", "Yaitska"));
+
+// const fullname1 = "Sascha";
+// const fullname2 = "Nata";
+
+// console.log(getFullname(fullname1, fullname2));
 
-const fullname1 = "Sascha";
-const fullname2 = "Nata";
 
-console.log(getFullname(fullname1, fullname2));
\ No newline at end of file
+// 2. second part
+
+const useFormalName = true;
+const gender = "female"
+
+function getFullname(firstname, surname) {
+    if (firstname === "" || firstname === Number) {
+        return "Input field filled out incorrectly";
+    } else if (useFormalName && gender === "female") {
+        return `Lady ${firstname} ${surname}`;
+    } else if (useFormalName && gender === "male") {
+        return `Lord ${firstname} ${surname}`;
+    } else if (useFormalName !== true) {
+       return firstname + " " + surname;
+    } else {
+        return "Try again."
+}
+};
```

Comment:
Also, you are right that in order to make this function work for both sexes we would have to add another parameter, maybe something like `isMale`, and if it is `true` add 'Lord', otherwise add 'Lady'. You do not have to add this to your function, it is wnough if you just write this into a comment. 😇